setup.py

```diff
@@ -8,7 +8,7 @@ def readfile(name):
 changes = readfile('CHANGES.rst')
 
 install_requires = [
-    'PasteDeploy >= 1.5.0',  # py3 compat
+    'PasteDeploy >= 2.0',  # py3 compat
```

Comment:
the pin should be adjusted to indicate why it's pinned to 2.0. Likely something along the lines of "default override order".